trySwift/UIImage+Downloader.swift

```diff
@@ -0,0 +1,27 @@
+//
```

Comment:
What's this for?

Reply:
It was an example to separate 3rd library from code so it's to be replaced easily anytime, I'll remove it